Rakefile

```diff
@@ -62,7 +62,7 @@ namespace :benchmark do
 
     corpus = File.expand_path(ENV["CORPUS"] || "samples")
 
-    require 'linguist/language'
+    require 'linguist'
```

Comment:
`require linguist/language` doesn't work for me either :grin:

We restructured some of the requires a while ago and I think this is just out-of-date code.